models/gyroscope/gyroscope_schema.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+
+models:
+
+  - name: gyroscope_trades
+
+    meta:
+
+      blockchain: ethereum, arbitrum, optimism, polygon
+      project: gyroscope
+      contributors: fmarrr
+
+    config:
+
+      tags: ["eclp", "gyroscope", "trades", "amm"]
+
+    description: trades on gyroscope E-CLP pools on all chains
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - token_bought_amount
+            - token_sold_amount
+            - tx_hash      
```

Comment:
no need for this test, as it's just a view

Reply:
the cross-chain views can leave it out, as `blockchain` is included in keys to help differentiate

Reply:
sorry, didn't get what you mean by 'cross-chain views can leave out' in this gyroscope_schema.yml. Mind to explain a bit please

Reply:
yep, no problem

i mean the `dbt_utils.unique_combination_of_columns` test applied in the schema yml file only needs applied to the upstream incremental spells, while the views built on top that union those together don't need to rerun the test as no data has changed outside of union'ing them together. the `blockchain` column within the union helps differentiate the project cross-chain 👍 

technically, you could leave this test there, but i guess my point is that it doesn't do anything additional for us. you could simply remove the test from the schema yml file, if it's a view and/or cross-chain union spell

Reply:
ah i see you removed, perfect!